src/index.js

```diff
@@ -85,8 +88,16 @@ class QrCodeScanner extends Component {
     qr.decode(imageData);
   }
 
+  bindVideoStream() {
+    if (typeof this.videoTag !== 'undefined') {
```

Comment:
I'm not sure if `typeof` should be the go to for this.
```suggestion
    if (this.videoTag != null) {
```
This will check for every kind of absent value.
[abstract equality operator](http://www.ecma-international.org/ecma-262/6.0/#sec-abstract-equality-comparison)

Reply:
Ah, alright. That actually seems way better!